### PR TITLE
Add AWS Kustomize manifests

### PR DESCRIPTION
**Pull request by Jeffwan**
**Which issue is resolved by this Pull Request:**
Resolves #https://github.com/kubeflow/kubeflow/issues/3151. This is a dependency task to be done to release 0.6 for aws.

**Description of your changes:**
Add AWS kustomize manifest to deprecate ksonnet package managers.

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate`
    3. `make test`

All tests passed. Also fix a istio test failure.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/221)

### CODE REVIEW

File: istio/istio/base/kustomization.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ vars:
   fieldref:
     fieldpath: data.clusterRbacConfig
 configurations:
- - params.yaml
```
Comment (Jeffwan) [inline istio/istio/base/kustomization.yaml]: This make params.yaml not populated in the test.go. Seems this repo does have test check. Not sure how does it check in the code base.

**Comment by Jeffwan** [comment]
Looks like an unrelated test case. Is it safe to remove ksonnet test cases in test-infra now?

**Comment by jlewi** [comment]
@Jeffwan I'll rubber stamp this PR; If you're fine with that just remove the hold. Otherwise if you want an actual review maybe wait for @mameshini or someone else familiar with AWS.

/lgtm
/approve

/hold

**Comment by Jeffwan** [comment]
@jlewi Thanks. I asked Kam for review and he will take a look at manifests and kfctl platform changes.

**Pull request merged**